librz/arch/function.c

```diff
@@ -310,23 +310,18 @@ RZ_API bool rz_analysis_function_rename(RzAnalysisFunction *fcn, const char *nam
  *
  * @sa rz_analysis_function_rename
  * */
-RZ_API const char *rz_analysis_function_force_rename(RzAnalysisFunction *fcn, const char *name) {
+RZ_API const char *rz_analysis_function_force_rename(RZ_NONNULL RzAnalysisFunction *fcn, RZ_NONNULL const char *name) {
```

Comment:
```suggestion
RZ_API RZ_BORROW const char *rz_analysis_function_force_rename(RZ_NONNULL RzAnalysisFunction *fcn, RZ_NONNULL const char *name) {
```

Reply:
This is already done [here](https://github.com/rizinorg/rizin/pull/4703/commits/057500dc42364527c5db3116e2e729d57d5109b3#diff-79ead514aa1e847124523b7f25695cb7e436de2494b5b09d7533a0ae1b3332baR313)